test/src/code_field/tabs_to_spaces_test.dart

```diff
@@ -0,0 +1,223 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_code_editor/src/code_field/text_editing_value.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+const _codeRowWithoutTabs = 'int a = 1;';
```

Comment:
`_codeLineWithoutTabs`
Although I don't see a point in testing a single line separately.
For code without tabs, you can just reuse `_codeWithDoubleSpaces` and have one less `const`.